src/sbus/router/sbus_router_handler.c

```diff
@@ -127,6 +128,11 @@ static void sbus_issue_request_done(struct tevent_req *subreq)
     DBusMessage *reply;
     errno_t ret;
 
+    /* This is a top level request. It can reply to more then one dp request
+     * therefore we need to set the id to 0 since it is not known at this
+     * moment. */
+    sss_chain_id_set(0);
```

Comment:
No need to restore chain_id at the end of this function?

Reply:
Not necessarily, but it is good for clarity and future proof. Fixed.